### PR TITLE
Minor changes 2.0.2

### PR DESCRIPTION
**Pull request by botzill**
Some test changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,3 @@
-# akamai-sensordata
-Generates Akamai x-acf-sensor-data header for all mobile application, accross all websites.
```
Comment (dshargorodsky0) [inline README.md:2]: Scam

**Review by dshargorodsky0 — COMMENTED** [review]
Scam

**Comment by ahubertus44** [comment]
Hi.

i need an Akamai Sensor Data Generator
Too bad this guy is scamming i was about to contact him and buy it ---
Do you have an alternative service actually?

**Comment by botzill** [comment]
Yes, 100% scam

**Pull request closed without merging**